src/content/docs/tutorial-customer-experience/intro-to-customer-experience.mdx

```diff
@@ -0,0 +1,52 @@
+metaDescription: "New Relic observability maturity series: this guide gives you a quality foundation for optimizing your web application's performance to improve your customer experience."
+redirects:
+  - /docs/new-relic-solutions/observability-maturity/qf-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/customer-experience/quality-foundation-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/customer-experience/customer-experience-quality-foundation-guide
+---
+
+import cxWhatYouCanMeasureNr from 'images/cx_diagram_what-you-can-measure-nr.webp'
+
+Digital customer experience refers to the quality of how your users interact with your service across all your digital touch points (web, mobile, and IoT). Because it has so many moving parts within the concept, it can be difficult to digest when taken as whole. You need to manage the availability of your service, its performance to ensure that it's usable, the quality of the content that gives users the information they need, and the relevance of your products and content so that users remain engaged.
+
+New Relic can help you make managing this much more approachable by measuring and improving performance in a way that better aligns to the end user experience. This guide covers the procedures that help you understand your digital customer experience in a meaningful way:
+
+* Look at customer experience in relation to:
+  * Global functions, such as search and login
+  * Lines of business
+  * Regions
+* Report back to business stakeholders on what they care about
+* Prioritize what to improve next
+
+<img
+  title="Digital Customer Experience (DCX): What you can measure with New Relic"
+  alt="Digital customer experience"
+  src={cxWhatYouCanMeasureNr}
+/>
+
+## Prerequisites
+
+Before you begin, you should have a few specific parts of our platform up and running. These include: 
+
+* [The Browser Pro agent version installed on all pages](/docs/browser/browser-monitoring/installation)
+* [SPA enabled for single page applications](/docs/new-relic-solutions/best-practices-guides/full-stack-observability/browser-monitoring-best-practices-guide/#how-to-do-it)
+* Synthetics monitors configured:
+  * [Ping monitors configured for anonymous users](/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors)
+  * [Scripted synthetics check configured for login flow](/docs/synthetics/synthetic-monitoring/using-monitors/store-secure-credentials-scripted-browsers-api-tests)
+  * Monitors should be configured to [test from all regions applicable to your users](/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors/#setting-location)
+  * Monitors should be configured for each domain and each login flow
+* [Data retention](/docs/data-apis/manage-data/manage-data-retention) for browser events greater than or equal to 2x an average sprint
+
+It's also helpful to have some basic knowledge of: 
```

Comment:
I assume if I've installed all of these things that I'd already have some basic knowledge of them... Maybe this section is unnecessary?

Reply:
Good callout, I'll check with the OMA crew. Thank you!